docs/pipeline-api.md

```diff
@@ -2,7 +2,7 @@
 ---
 title: Pipeline API
 linkTitle: Pipeline API
-weight: 404
+weight: 1000
```

Comment:
why are you changing the weight? please revert this to 404

https://github.com/tektoncd/pipeline/pull/6200 fixes the script